damn-fast-priority-queue/test.lisp

```diff
@@ -122,7 +127,7 @@
 
 (defun test-dequeue-and-peek (queue list)
   (let ((counter (q:size queue)))
-    (dotimes (i (length list))
+    (dolist (i (sort list '<))
```

Comment:
This mutates the input list which might cause confusion. Change `list` to `(copy-list list)`.